src/components/ToDoContainer/NewTodoForm.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const NewTodoForm = (props) => {
```

Comment:
Instead of passing `props` as an argument here and breaking it down, use destructuring (kind of like pattern matching from CS51) to pass in `{formSubmitted, newTodoChanged, newTodo}`, etc. so that you don't always have to access them as properties of `props`.